starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy

```diff
@@ -0,0 +1,14 @@
+package io.micronaut.starter.options
+
+import spock.lang.Specification
+
+class BuildToolSpec extends Specification {
+
+    void "valueGradle return Groovy and Kotlin DSL"() {
+        BuildTool.valuesGradle().stream().allMatch { it.isGradle() }
+        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE_KOTLIN }
+        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE }
+        [BuildTool.GRADLE, BuildTool.GRADLE_KOTLIN].size() == BuildTool.valuesGradle().size()
+    }
```

Comment:
```suggestion
    void "valueGradle return Groovy and Kotlin DSL"() {
        expect:
        BuildTool.valuesGradle().stream().allMatch { it.isGradle() }
        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE_KOTLIN }
        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE }
        [BuildTool.GRADLE, BuildTool.GRADLE_KOTLIN].size() == BuildTool.valuesGradle().size()
    }
```

Reply:
Otherwise, this isn't run as a test

Reply:
Fixed in https://github.com/micronaut-projects/micronaut-starter/pull/1657/commits/dfddd837e65952b260aaf4838bca0a2d70b81424

Reply:
good catch